applications/plugins/SofaPardisoSolver/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ find_package(OpenMP REQUIRED)
 set(LINKER_DEPENDENCIES ${LINKER_DEPENDENCIES} SofaBase SofaGeneral)
 set(COMPILATION_FLAGS -DSOFA_BUILD_SOFAPARDISOSOLVER )
 
-set ( PARDISO_LIB "" CACHE PATH "set to Pardiso library downloaded from the web" )
+set ( PARDISO_LIB "" CACHE FILE "set to Pardiso library downloaded from the web" )
```

Comment:
The cmake is not looking inside the "extlibs" to find the library while it should (well, we got the feeling it should look there automatically when we read the README.md)